docs/Installation.md

```diff
@@ -227,15 +227,39 @@ Make sure Zonemaster-GUI is properly installed.
 3. Verify that when you mouse over this text the versions of the following
    Zonemaster components are shown: Backend, Engine and GUI.
 
-
-
 ## What to do next?
 
  * For a JSON-RPC API, see the Zonemaster::Backend [JSON-RPC API] documentation.
  * For a command line interface, follow the [Zonemaster::CLI installation] instruction.
  * For a Perl API, see the [Zonemaster::Engine API] documentation.
  * For Https, see [Let's Encrypt / Certbot].
 
+## Serving the GUI and API from a custom base url
+
+In some cases you may want to customize the application to change the base URL
+from wich the GUI is served. To achieve that you will need to change the
+`<base>` in the `index.html`.
+
+Locate the line `<base href="/">` and replace the `href` value with the path you
+want to server the GUI from, e.g. `<base href="/zonemaster/">`. Don't forget the
+trailing `/`.
```

Comment:
In a default installation this is `/var/www/html/zonemaster-web-gui/dist/index.html`, which should be stated.

Reply:
done